mon-pix/tests/acceptance/challenge-qroc_test.js

```diff
@@ -307,32 +310,6 @@ describe('Acceptance | Displaying a QROC challenge', () => {
 
     });
 
-    describe('When challenge is already answered', () => {
-      beforeEach(async () => {
-        // given
-        server.create('answer', {
-          value: 'Reponse',
-          result: 'ko',
-          assessment,
-          challenge: qrocChallenge,
-        });
-
-        // when
-        await visit(`/assessments/${assessment.id}/challenges/${qrocChallenge.id}`);
-      });
-
-      it('should set the input value with the current answer and propose to continue', async () => {
-        // then
-        expect(find('.challenge-response__proposal').value).to.equal('Reponse');
-        expect(find('.challenge-response__proposal').disabled).to.be.true;
-
-        expect(find('.challenge-actions__action-continue')).to.exist;
-        expect(find('.challenge-actions__action-validate')).to.not.exist;
-        expect(find('.challenge-actions__action-skip-text')).to.not.exist;
-
-      });
-    });
-
```

Comment:
**question:** Ce cas fonctionnel n'existe plus du tout maintenant que l'on a des challenges number ? Que se passe t'il si on revient en arrière ?

Reply:
On a bien le test `When challenge is already answered` dans les tests d'acceptance du QROC. Ce qui est retiré ici est un doublon du fichier